src/status_im/events.cljs

```diff
@@ -163,8 +163,10 @@
  (fn [cofx _]
    (multiaccounts.logout/logout cofx)))
 
+
```

Comment:
why empty lines?

Reply:
No reason. I must have added something here and forgot to delete the empty line. Will fix it.